src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -54,16 +73,42 @@ export default {
 		};
 	},
 	async mounted() {
-		this.allFacets = await fetchLoanFacets(this.apollo);
+		this.allFacets = await fetchLoanFacets(this.apollo, true);
 		await this.fetchFilterData(this.flssLoanSearch);
-	},
-	computed: {
-		totalCount() {
-			return 0;
-		},
+		const { loans, totalCount } = await runLoansQuery(
+			this.apollo,
+			{ ...this.flssLoanSearch, ...this.loanSearchState },
+			FLSS_ORIGIN_CATEGORY
```

Comment:
Is this the correct query context? It would be good to ask the data team what context we should use for this new page, if we haven't already.

Reply:
We have a meeting with the data team to check this and any other requirements from them